rotkehlchen/chain/optimism/proxies_inquirer.py

```diff
@@ -0,0 +1,30 @@
+from typing import TYPE_CHECKING, Optional
+from rotkehlchen.chain.evm.proxies_inquirer import EvmProxiesInquirer
+from rotkehlchen.types import ChecksumEvmAddress
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.optimism.node_inquirer import OptimismInquirer
+
+
+class OptimismProxiesInquirer(EvmProxiesInquirer):
+    """Class to retrive information about DSProxy for defi addresses."""
+
+    def __init__(
+            self,
+            optimism_inquirer: 'OptimismInquirer',
+    ) -> None:
+        super().__init__(optimism_inquirer)
+        self.optimism = optimism_inquirer
+        self.dsproxy_registry = self.optimism.contracts.contract('DS_PROXY_REGISTRY')
+
+    def get_account_proxy(self, address: ChecksumEvmAddress) -> Optional[ChecksumEvmAddress]:
+        return None
+
+    def get_accounts_proxy(
+        self,
+        addresses: list[ChecksumEvmAddress],
+    ) -> dict[ChecksumEvmAddress, ChecksumEvmAddress]:
+        return {}
+
+    def get_accounts_having_proxy(self) -> dict[ChecksumEvmAddress, ChecksumEvmAddress]:
+        return {}
```

Comment:
why leave those unused?

Reply:
Just for now. Because I think it is better to implement the optimism-specific methods in a different PR. Will add a TODO in the code